exercises/secret-handshake/introduction.md

```diff
@@ -0,0 +1,7 @@
+# Introduction
+
+You are starting a secret coding club with some friends and friends-of-friends.
+Not everyone knows each other, so you and your friends have decided to create a secret handshake that you can use to recognise that someone is a member.
```

Comment:
I notice that in https://github.com/exercism/problem-specifications/pull/2220, the American spelling of "favorite" is used. But in this text, the British spelling of "recognise" is used. From this it looks like the rule is: It's okay for all words in an exercise to use the American spelling, or all words in an exercise to use the British spelling, as long as each exercise is internally consistent (a single exercise should not mix spellings). Did I get it right?

Reply:
We should be using only American English spellings on Exercism—we tend to accidentally mix things (and miss things), because we're a mixed team, some of whom write British English when not doing official Exercism stuff.

So it should be `recognize` here.

Reply:
```suggestion
Not everyone knows each other, so you and your friends have decided to create a secret handshake that you can use to recognize that someone is a member.
```